python/orca/dev/example/run-example-test-ray.sh

```diff
@@ -52,6 +52,9 @@ python ${BIGDL_ROOT}/python/orca/example/ray_on_spark/rllib/multi_agent_two_trai
 now=$(date "+%s")
 time5=$((now-start))
 
+python_version=$(python --version | awk '{print$2}')
+if [ $python_version == 3.7.10 ];then
```

Comment:
We specify the python version when [setup-env](https://github.com/intel-analytics/BigDL/blob/main/.github/actions/orca/setup-env/setup-orca-python-py37-basic/action.yml#L9).